run_tests.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash -ex
 
 export PIPENV_IGNORE_VIRTUALENVS=1
+set +x  # Keep the API key out of execution logs
+export DEPSDEV_API_KEY="$(gcloud --project oss-vdb secrets versions access latest --secret depsdotdev-key)"
+test -z "$DEPSDEV_API_KEY" && echo "No API key for deps.dev" && exit 1
```

Comment:
Can we make this optional and runnable by external/non-maintainer people?

Otherwise we'd also have to fix up https://github.com/google/osv.dev/blob/master/CONTRIBUTING.md#running-tests to say that it's only runnable for maintainers.

Reply:
Done in a40a406de8e0cd31f3c5eee473ddf603f341c2f1